src/controllers/TalksController.php

```diff
@@ -289,6 +367,12 @@ public function createTalkAction(Request $request, PDO $db)
         $tz = new DateTimeZone($event['tz_continent'] . '/' . $event['tz_place']);
         $talk['date'] = (new DateTime($start_date, $tz))->format('U');
 
+        $event_start_date = (new DateTime($event['start_date']))->format('U');
```

Comment:
I've just had a deeper look at this. It looks like the Server-Timezone is irrelevant, as the date-fields contain the offset so the Server-Timezone is ignored on DateTime-object creation.

So no ticket needed.